src/components/MainCard.tsx

```diff
@@ -102,7 +102,8 @@ const MainCard: React.FC<MainCardProp> = forwardRef(
         )}
 
         {/* card content */}
-        {content && <CardContent sx={contentSX}>{children}</CardContent>}
+        {/* id is used to select container, to reset its scroll value in the checkout page */}
```

Comment:
Same thing. Comment explains something that is not clear from the code, and can't be clarified.